client/main.lua

```diff
@@ -24,193 +23,219 @@ local currentGear = {
 local function callCops()
     local call = math.random(1, 3)
     local chance = math.random(1, 3)
-    local ped = PlayerPedId()
-    local coords = GetEntityCoords(ped)
+    local coords = GetEntityCoords(cache.ped)
+
     if call == chance then
         TriggerServerEvent('qb-diving:server:CallCops', coords)
     end
 end
+
 local function deleteGear()
 	if currentGear.mask ~= 0 then
         DetachEntity(currentGear.mask, 0, 1)
         DeleteEntity(currentGear.mask)
+
 		currentGear.mask = 0
     end
+
 	if currentGear.tank ~= 0 then
         DetachEntity(currentGear.tank, 0, 1)
         DeleteEntity(currentGear.tank)
+
 		currentGear.tank = 0
 	end
-
 end
+
 local function gearAnim()
-    RequestAnimDict("clothingshirt")
-    while not HasAnimDictLoaded("clothingshirt") do
-        Wait(0)
-    end
-	TaskPlayAnim(PlayerPedId(), "clothingshirt", "try_shirt_positive_d", 8.0, 1.0, -1, 49, 0, 0, 0, 0)
+    lib.requestAnimDict("clothingshirt")
+
+	TaskPlayAnim(cache.ped, "clothingshirt", "try_shirt_positive_d", 8.0, 1.0, -1, 49, 0, 0, 0, 0)
```

Comment:
```suggestion
    lib.requestAnimDict("clothingshirt")
    TaskPlayAnim(cache.ped, "clothingshirt", "try_shirt_positive_d", 8.0, 1.0, -1, 49, 0, 0, 0, 0)
```

Just some leftover whack indentation